torch/_inductor/scheduler.py

```diff
@@ -405,6 +416,59 @@ def can_inplace(self, read_dep: dependencies.MemoryDep):
             return read_dep.index == write_dep.index and read_dep.size == write_dep.size
         return False
 
+    def reorder_loop(
+        self, reorder: Callable[[sympy.Expr], sympy.Expr], ndim: int
+    ) -> Optional["SchedulerNode"]:
+        """Returns a new node which represents the same loop iterated in a new order
+
+        reorder is a function that takes indices in the new order and
+        translates them into the indices expected by the underlying loop. e.g.
+
+            reorder = lambda index: index[1], index[0]
```

Comment:
Could this just be a dict/list?  Is a callable actually needed?

Reply:
Reverted back to a list.